indigo/distribution.yaml

```diff
@@ -5951,10 +5951,20 @@ repositories:
       type: git
       url: https://github.com/ros-geographic-info/open_street_map.git
       version: master
+    release:
+      packages:
+      - osm_cartography
+      - route_network
+      - test_osm
```

Comment:
You mean the test package? Yes to that.

But there's a bug in one test case, so I'll need to release this again with that fix. Just waiting for CI to run. I'll close this PR now.